### ISSUE TITLE
Compatibility with Opsdroid > v0.19.0

### ISSUE THREAD

**Issue by oleg-fiksel** [opsdroid/opsdroid-homeassistant#20]
opsdroid-homeassistant doesn't work on Opsdroid v0.20.0. 😢 

## Steps to reproduce

* Use Opsdroid v0.20.0 (I used Docker container)
* Install opsdroid-homeassistant: `pip install opsdroid-homeassistant==0.1.8`
* Create a sample skill:
```
from asyncio import sleep
from opsdroid_homeassistant import HassSkill, match_hass_state_changed

class MotionLights(HassSkill):

    @match_hass_state_changed("binary_sensor.drive", state="on")
    async def motion_lights(self, event):
        """Turn the outside light on with motion if after sunset."""
        if await self.sun_down():
            await self.turn_on("light.drive")
            await sleep(60)
            await self.turn_off("light.drive")
```
* Start Opsdroid: `opsdroid start`

## Expected result

* Skill is loaded and Opsdroid starts

## Current result

```
INFO opsdroid.logging: ========================================
INFO opsdroid.logging: Started opsdroid v0.20.0.
WARNING opsdroid.loader: No databases in configuration. This will cause skills which store things in memory to lose data when opsdroid is restarted.
Traceback (most recent call last):
  File "/usr/local/bin/opsdroid", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/opsdroid/cli/start.py", line 43, in start
    opsdroid.run()
  File "/usr/local/lib/python3.8/site-packages/opsdroid/core.py", line 166, in run
    self.sync_load()
  File "/usr/local/lib/python3.8/site-packages/opsdroid/core.py", line 215, in sync_load
    self.eventloop.run_until_complete(self.load())
  File "/usr/local/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.8/site-packages/opsdroid/core.py", line 224, in load
    self.setup_skills(self.modules["skills"])
  File "/usr/local/lib/python3.8/site-packages/opsdroid/core.py", line 286, in setup_skills
    skill_obj = func(self, skill["config"])
  File "/usr/local/lib/python3.8/site-packages/opsdroid_homeassistant/skill/__init__.py", line 44, in __init__
    [self.hass] = [
ValueError: not enough values to unpack (expected 1, got 0)
```

My aim is to run this skill on Opsdroid v0.20.0: https://gitlab.com/olegfiksel/home-assistant-bot/-/tree/5-upgrade-to-opsdroid-v0-20-0

**Comment by oleg-fiksel** [comment]
Seems that `self.opsdroid.connectors` at https://github.com/opsdroid/opsdroid-homeassistant/blob/master/opsdroid_homeassistant/skill/__init__.py#L44 is an empty list.

**Comment by jacobtomlinson** [comment]
Sorry about this.

Could you also share your configuration?

**Comment by oleg-fiksel** [comment]
Here is the config I was using for the simple test:

```
welcome-message: false

connectors:
  homeassistant:
    url: $HOME_ASSISTANT_URL
    token: $HOME_ASSISTANT_TOKEN

  matrix:
    mxid: $MATRIX_MXID
    password: $MATRIX_PASSWORD
    homeserver: $MATRIX_SERVER
    rooms:
        main:
          alias: $MATRIX_ROOM_MAIN
          send_m_notice: True
        debug:
          alias: $MATRIX_ROOM_DEBUG
          send_m_notice: True

skills:
  home-assistant-bot:
    path: /skills/home-assistant-bot
```

For the home-assistant-bot I'm using the following config: https://gitlab.com/olegfiksel/home-assistant-bot/-/blob/5-upgrade-to-opsdroid-v0-20-0/sample-config.yaml

**Comment by davdr** [comment]
@oleg-fiksel @jacobtomlinson PR #21 explains the root cause and solves the problem.

**Comment by oleg-fiksel** [comment]
I can confirm, https://github.com/opsdroid/opsdroid-homeassistant/pull/21/commits/6296e0dfea4bcd23d5a637de0b9433a1e6e5a60a fixes this.
Thank you @davdr ! ❤️

**Comment by davdr** [comment]
PR #21 updated: now the unit tests are also fixed for Opsdroid 0.20+ compatibility (and they pass).
